src/components/RelicModal.jsx

```diff
@@ -303,7 +478,344 @@ export default function RelicModal(props) {
     (option?.label ?? '').toLowerCase().includes(input.toLowerCase())
 
   const plusThree = () => {
-    relicForm.setFieldValue("enhance", Math.min(relicForm.getFieldValue("enhance") + 3, 15));
+    relicForm.setFieldValue('enhance', Math.min(relicForm.getFieldValue('enhance') + 3, 15))
+  }
+
+  const [wearerID, setWearerID] = useState('None')
+
+  const [hide0, setHide0] = useState('inline')
+
+  const [hide1, setHide1] = useState('inline')
+
+  const [hide2, setHide2] = useState('inline')
+
+  const [hide3, setHide3] = useState('inline')
+
+  const [button0low, setButton0low] = useState(undefined)
+
+  const [button0mid, setButton0mid] = useState(undefined)
+
+  const [button0high, setButton0high] = useState(undefined)
+
+  const [button1low, setButton1low] = useState(undefined)
+
+  const [button1mid, setButton1mid] = useState(undefined)
+
+  const [button1high, setButton1high] = useState(undefined)
+
+  const [button2low, setButton2low] = useState(undefined)
+
+  const [button2mid, setButton2mid] = useState(undefined)
+
+  const [button2high, setButton2high] = useState(undefined)
+
+  const [button3low, setButton3low] = useState(undefined)
+
+  const [button3mid, setButton3mid] = useState(undefined)
+
+  const [button3high, setButton3high] = useState(undefined)
```

Comment:
I dont think we need separate states for each of these, can just be a single object